astropy/modeling/tests/test_functional_models.py

```diff
@@ -153,6 +153,16 @@ def test_Shift_inverse():
     assert_allclose(m.inverse(m(6.789)), 6.789)
 
 
+def test_Scale_fittable():
+    m = models.Scale(1.2)
+    assert(m.fittable==True)
+
+
+def test_Shift_fittable():
+    m = models.Shift(1.2)
+    assert(m.fittable==True)
+
+
```

Comment:
I was thinking more like tests that do actual fitting and then check whether the fitted value is correct. For example, fitting a model that is shifted by 1.2 should return 1.2, or something along that line.